schema/schema.sql

```diff
@@ -0,0 +1,66 @@
+DROP TABLE IF EXISTS purchase;
+DROP TABLE IF EXISTS item_genre_assignment;
+DROP TABLE IF EXISTS genre;
+DROP TABLE IF EXISTS item;
+DROP TABLE IF EXISTS artist;
+DROP TABLE IF EXISTS format;
+DROP TABLE IF EXISTS country;
+DROP TABLE IF EXISTS subscriber;
+
+
+CREATE TABLE subscriber (
+    subscriber_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    name VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE country (
+    country_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(60) UNIQUE NOT NULL
+);
+
+CREATE TABLE format (
+    format_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(20) UNIQUE NOT NULL
+);
+
+CREATE TABLE artist (
+    artist_id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(255) UNIQUE NOT NULL,
+    url VARCHAR(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE item (
+    item_id SMALLINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    track_title VARCHAR(255) NOT NULL,
+    album_title VARCHAR(255) NOT NULL, 
```

Comment:
These can be NULL. Instead of this we need to add a check constraint that both cannot be NULL

Reply:
Noted. I've applied changes to the constraints regarding that table to ensure that it works like this instead.